tools/glb-viewer/src/viewer.js

```diff
@@ -308,8 +309,10 @@ Object.assign(Viewer.prototype, {
     }
 });
 
-var viewer;
+/* eslint-disable no-unused-vars */
 
 var main = function () {
-    viewer = new Viewer(document.getElementById("application-canvas"));
+    new Viewer(document.getElementById("application-canvas"));
```

Comment:
I'm not a big fan of not assigning the result of calling a constructor to anything.

Reply:
Try telling our lint overlord so! Would you prefer I disable that check here instead?

Reply:
Oh I see it's already in that block (due to main being called from index.html).

Reply:
it's back!